src/include/migraphx/op/scatternd_op.hpp

```diff
@@ -121,7 +121,8 @@ struct scatternd_op : op_name<Derived>
                 auto k             = indices_shape.lens().back();
                 auto q             = indices_shape.ndim();
                 auto r             = dyn_out.computed_shape.ndim();
-                par_for(updates_shape.elements(), [&](const auto i) {
+                for(auto i = 0u; i < updates_shape.elements(); ++i)
```

Comment:
What is the reason for removing parallelism here?

Reply:
To produce correct results in case of duplicate indices. A more complex solution could have been implemented, but I don't think the reference implementation warrants it